packages/opentelemetry-sdk-trace-web/src/utils.ts

```diff
@@ -266,6 +264,19 @@ function getByteLength(s: string): number {
   return TEXT_ENCODER.encode(s).byteLength;
 }
 
+function getFormDataSize(formData: FormData): number {
+  let size = 0;
+  for (const [key, value] of formData.entries()) {
+    size += key.length;
+    if (value instanceof Blob) {
+      size += value.size;
+    } else {
+      size += value.length;
+    }
+  }
+  return size;
+}
```

Comment:
This is an improvement over the old implementation (which was stringifying the FormData, and that wouldn't actually stringify file objects correctly), but it still doesn't calculate the _exact_ FormData size. Browsers will serialize FormData in their own way (ex including a separator for each of the fields, etc.) 

I will keep investigating for more accurate approaches.

Reply:
Yeah, this seems good enough for now, and maybe if anyone is sufficiently motivated they can contribute the changes to calculate it more precisely for their platform 😬